package.json

```diff
@@ -1,77 +1,81 @@
 {
-	"name": "cryptly",
-	"version": "0.1.14",
-	"description": "Fullstack encryption library.",
-	"author": "PayFunc",
-	"license": "MIT",
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/payfunc/cryptly"
-	},
-	"bugs": {
-		"url": "https://github.com/payfunc/cryptly/issues"
-	},
-	"homepage": "https://payfunc.com/open-source/cryptly",
-	"private": false,
-	"main": "dist/index.js",
-	"typings": "dist/index.d.ts",
-	"browser": {
-		"./dist/crypto": "./dist/crypto.browser"
-	},
-	"git": {
-		"tagName": "v${version}"
-	},
-	"github": {
-		"release": true
-	},
-	"jest": {
-		"globals": {
-			"ts-jest": {
-				"tsConfig": "tsconfig.test.json"
-			}
-		},
-		"testEnvironment": "node",
-		"testRegex": "((\\.|/)(test|spec))(\\.|\\/.+)(jsx?|tsx?)$",
-		"testPathIgnorePatterns": [
-			"node_modules/",
-			"dist/"
-		],
-		"collectCoverageFrom": [
-			"**/*.{ts,tsx,js,jsx}",
-			"!**/node_modules/**",
-			"!**/dist/**"
-		],
-		"preset": "ts-jest",
-		"testMatch": null
-	},
-	"scripts": {
-		"dev": "watch 'npm run build' .",
-		"build": "tsc -p .",
-		"lint": "eslint '**/*.{js,ts,tsx}'",
-		"fix": "eslint '**/*.{js,ts,tsx}' --fix",
-		"test": "jest",
-		"test:watch": "watch jest",
-		"prepare": "npm run build",
-		"clean": "rm -rf dist node_modules coverage"
-	},
-	"optionalDependencies": {
-		"node-webcrypto-ossl": "^2.1.2"
-	},
-	"devDependencies": {
-		"@types/jest": "^26.0.14",
-		"@types/node": "^14.11.10",
-		"@typescript-eslint/eslint-plugin": "^4.4.1",
-		"@typescript-eslint/parser": "^4.4.1",
-		"babel-core": "^6.26.3",
-		"babel-jest": "^26.6.0",
-		"eslint": "^7.11.0",
-		"eslint-config-standardize": "^0.7.1",
-		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20220323",
-		"eslint-plugin-simple-import-sort": "^7.0.0",
-		"jest": "^26",
-		"prettierx": "github:utily/prettierx#utily-20220323",
-		"ts-jest": "^26.4.1",
-		"typescript": "^4.0.3",
-		"watch": "^1.0.2"
-	}
+  "name": "cryptly",
+  "version": "0.2.0",
```

Comment:
Cryptly has a parity, this version number doesn't make sense because of it. I recommend to bump major version instead.

Reply:
And what is the reason we don't continue in the same way as we normally do with (master-0.0 = 0.2k.m) and (master = 0.2k+1.m) ?

Reply:
If you mean use 0.2.x (parity) and 0.3.x (master) for next iterations I am okay with that

Reply:
Yeah, this bump will create version `0.3.1`.
And I will bump parity to `0.1.0` which will create version `0.2.1`

So the structure is `x.2k.y` for parity and `x.2k+1.y` for non parity, where k, x, y ∈ℕ.

Reply:
No, there is no automatic bumping of the minor value, you have to do that one manually

Reply:
Ah, then I need to change :)